backend/src/Manager/Order/OrderManager.php

```diff
@@ -19,6 +19,7 @@
 use App\Manager\Captain\CaptainManager;
 use DateTime;
 use App\Request\Order\OrderUpdateCaptainOrderCostRequest;
+use App\Manager\OrderLogs\OrderLogsManager;
 
 class OrderManager
```

Comment:
Ehm Ehm:
please move this logic "create logs" to its own service.
then call it from order service.
this approach kills the "single responsibility" principle